macosx/Collection+Additions.swift

```diff
@@ -0,0 +1,49 @@
+// This file Copyright © 2022-2023 Transmission authors and contributors.
+// It may be used under the MIT (SPDX: MIT) license.
+// License text can be found in the licenses/ folder.
+
+import Foundation
+
+extension Collection where Self: RandomAccessCollection {
+    /// Returns a copy of the receiving array sorted as specified by a given array of sort descriptors.
+    @inlinable
```

Comment:
afaik `inlinable` does something only if it's in a public interface of a module that is consumed from another module, so here it does nothing, with one module the compiler will make the decision itself and we can't force it.

Reply:
As an "addition", it's meant to be extractable as an independent module eventually. So I prefer to keep @inlinable for now.